src/index.sass

```diff
@@ -59,6 +59,20 @@ button
     background: $primary-color
     color: #fff
 
+#tab-bar
+    display: flex
+    flex-direction: row
+    justify-content: space-around
+    background-color: $primary-color
+    padding: 1rem
+    position: fixed
+    bottom: 0;
```

Comment:
I don't know what the red means but the css should be consistent with the usage of ; or not ;
![image](https://user-images.githubusercontent.com/5315633/80696343-9031b580-8ad7-11ea-91a2-7e7d41cb20e0.png)